content/applications/sales/sales/send_quotations/optional_products.rst

```diff
@@ -1,65 +1,89 @@
 
-.. note::
-   By clicking on the **Customer Preview** button, you will be able to see what your customers
-   will have as possibilities after opening a quotation from their received email.
+.. tip::
+   Click the :guilabel:`Customer Preview` button to reveal a preview of the quotation customers
+   would receive via email, along with the optional products they have a chance of adding to their
+   order.
 
-   .. image:: optional_products/optional_products_3.png
+   .. image:: optional_products/optional-products-checkout.png
       :align: center
-      :class: img-thumbnail
-      :alt: Preview your quotations on Odoo Sales
+      :alt: Preview your quotations on Odoo Sales.
 
-In practice, your customers will be able to add different optional products to their order by
-using associated carts, with a user-friendly layout. More than that, if a customer selects all the
-optional products suggested, these additional items will automatically fill in the quotation
-managed by the salesman.
+In practice, customers are able to add different optional products to an order by clicking the
+:guilabel:`🛒 (shopping cart)` icon, located to the right of the optional product line. If a
+customer selects all the optional products suggested, these additional items are automatically
+filled in on the quotation that is managed by the salesperson.
 
-.. image:: optional_products/optional_products_4.png
+.. image:: optional_products/optional-products-add-to-cart.png
    :align: center
-   :class: img-thumbnail
-   :alt: How to select optional products on Odoo Sales
+   :alt: How to select optional products on Odoo Sales.
 
-Like this, salespeople will see each movement made by the customer and tracking the order will be
-all the better.
+When the customer adds an optional product(s) to an order, the salesperson is instantly notified
+about the change, along with any other change the customer makes to an order. This allows
+salespeople to stay up-to-date with everything related to an order in the backend of the *Sales*
+application.
 
-.. image:: optional_products/optional_products_5.png
+.. image:: optional_products/optional-products-order-lines.png
    :align: center
-   :class: img-thumbnail
-   :alt: How to follow each movement made by your customers on Odoo Sales
+   :alt: How to follow each movement made by your customers on Odoo Sales.
 
-Add optional products to your quotation templates
-=================================================
+Optional products on quotation templates
+========================================
 
-Be sure to check out our documentation about :doc:`quote_template` to understand how you can
-enable, create, design and manage your own quotation templates before reading this part.
+Be sure to review the documentation about
+:doc:`/applications/sales/sales/send_quotations/quote_template` to better understand how quotation
+templates can be enabled, created, designed, and managed before reading the following information.
```

Comment:
optional: less wordy
would consider putting this entire sentence in a **note** or **tip** admonition block
```suggestion
:doc:`/applications/sales/sales/send_quotations/quote_template` to better understand how quotation templates
work before reading the following information.
```